src/components/send/SendAssetForm.js

```diff
@@ -84,8 +84,12 @@ export default function SendAssetForm({
 
   const { colors } = useTheme();
 
-  const address = selected?.address;
-  const colorForAsset = useColorForAsset({ address });
+  const address = selected?.mainnet_address || selected?.address;
+
+  let colorForAsset = useColorForAsset({ address });
+  if (isNft) {
+    colorForAsset = colors.appleBlue;
+  }
```

Comment:
what about 
```
const colorForAsset = isNft ?  colors.appleBlue : useColorForAsset({ address });
```
instead of assigning and reassigning if `isNft`

Reply:
ja, like this https://github.com/rainbow-me/rainbow/pull/2116/files#diff-ef3c2612d4b13fa4746945abeacdf6102915e08ea5ea5f9cee7be0a5a15ef7b5R14

Reply:
You can't call hooks conditionally!